docs/user-guide/ze-profiles.md

```diff
@@ -21,28 +30,34 @@ Create a team configuration file.
 6. Edit the config file to include the host information and save the file.
-7. Refresh Zowe Explorer by either clicking the button in the notification message shown after creation, `alt+z`, or the `Zowe Explorer: Refresh Zowe Explorer` command palette option.
+7. Refresh Zowe Explorer by either clicking the button in the notification message shown after creation, `alt` + `z`, or the `Zowe Explorer: Refresh Zowe Explorer` command palette option.
 
-Your team configuration file appears either in your `.zowe` folder if you choose the global configuration file option, or in your workspace directory if you choose the project-level configuration file option. The notification message that shows in VS Code after config file creation will include the path of the file created.
+    Your team configuration file appears either in your `.zowe` folder if you choose the global configuration file option, or in your workspace directory if you choose the project-level configuration file option. The notification message that displays in VS Code after the configuration file creation includes the path of the file created.
 
 ### Managing profiles
 
-You can edit your project-level or global configuration files.
-
-**Follow these steps**:
+Change profile validations and edit the profiles in your project or global configuration files.
 
 1. Right-click on your profile.
-2. Select the **Add**, **Update**, or **Delete Profile** options to edit the zowe config file in place.
+2. Select the **Manage Profile** option to choose from several authentication and profile management actions for the credentials detected in your Zowe Explorer session.
 
-   **Tip:** Use the Intellisense prompts if you need assistance with filling parameters in the file.
+    Authentication options display according to the detected credentials:
 
-3. Save the config file.
+    - **Add Credentials** to store a username and password in **WHERE**
+    - **Update Credentials** to update the username and password stored in **WHERE**
+    - **Log in to authentication service** to obtain a new authentication token when the token in the profile is no longer valid or is missing
+    - **Log out of authentication service** to invalidate the token in the profile so a valid token is not stored
 
-4. Refresh the view by clicking the refresh icon in the Data Sets, USS, or Jobs view.
+    Profile management options displays for specific profile actions:
 
-   Alternatively, press F1 to open the command palette, type and execute the **Zowe Explorer: Refresh Zowe Explorer** option.
+    - **Disable/Enable Profile Validation** to disable or enable validation of access to z/OSMF
+    - **Edit Profile** to update profile information in an **Editor** tab
+    - **Hide Profile** to hide the profile name from the tree view
+    - **Delete Profile** to remove the profile from the configuration file
```

Comment:
is "configuration file" correct here?

Reply:
This is an odd one: `Delete Profile` works the same as `Edit Profile`, so it just opens the configuration file in an editor tab. Not sure what we should say here.

Reply:
@zFernand0 -- what are your thoughts?

Reply:
We can say something similar to the Edit profile option.

- `to remove the profile information in an Editor tab`


---

However, in reality, we should consider removing this in V2 (and Vnext)